### DISCUSSION TITLE
Amazon icons

### DISCUSSION THREAD

**Discussion by manriqueply** [Mariusthvdb/custom-icons#40] [Icon suggestions/requests]
Im pretty new to the icon creation, but i manage to create some for my lovelace. Hope it helps.

![Echo dot v2](https://user-images.githubusercontent.com/75438978/187580939-3d346345-1c9d-4aff-9559-a80f76bdf5fa.svg)
![Echo](https://user-images.githubusercontent.com/75438978/187580594-450c7e57-1850-4faf-bcad-90c65c828c9a.svg)
![FireTV](https://user-images.githubusercontent.com/75438978/187580595-2ab989fb-4a04-463e-94db-ab07b8385cc6.svg)

**Comment by Mariusthvdb** [comment]
Ill ask @arallsopp to have look here, to try and follow MDI specs as much as possible.
You'd best create a PR, so we can comment and change where needed

**Reply by arallsopp** [reply]
Hi there. I create and maintain [hass-hue-icons](https://github.com/arallsopp/hass-hue-icons) for Philips Hue devices and am a little guilty of putting the overspill of icons that don't quite belong there into this repo. As such, I don't really follow the MDI guidelines, which probably makes @Mariusthvdb's job a little murkier.

The rules I use are based upon a 24x24px canvas and stipulate
- no line is allowed to go below 1px width.
- no line is allowed within 1px of the border.
- no white space is allowed to go below 1px width.

I'm on a fairly high def retina screen with very good sub pixel aliasing that allows icons with far smaller detail to appear perfectly well, but I try to to keep to these standards so that *everyone* can use my icons. 

It might just be a case of drawing a vector curve over the icons you have, adding suitable line-weight to it, outlining the stroke and adding it to the path, then subtracting that overlay from the original.
